frontend/pages/hosts/details/DeviceUserPage/ManualEnrollMdmModal/ManualEnrollMdmModal.tsx

```diff
@@ -97,6 +97,7 @@ const ManualEnrollMdmModal = ({
               </Button>
             )}
           </li>
+          <li>Click on the profile you just downloaded.</li>
```

Comment:
@jacobshandling nice catch! Does the user have to single-click or double-click? Also, can you please attach a screenshot of the UI change? Thanks :)

Reply:
Good point – they can single click if accessing their downloads (or any) folder from the dock via the accordion feature, double-click if it's via Finder. How about "Open the profile you just downloaded." ? That would cover command line users as well.

Reply:
@jacobshandling "Open the profile you just downloaded." sounds good to me